starter-code/index.html

```diff
@@ -0,0 +1,261 @@
+            <div class="carousel-caption d-none d-md-block ">
+              <h5>ETIAM MAURIS TORTOR, PHARETRA QUIS LOBORTIS IN, PHARETRA IN DIAM</h5>
+              <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+              <button type="button" class="btn btn-outline-light">Read More</button>
+            </div>
+          </div>
+        </div>
+       
+        </div>
+        <a class="carousel-control-prev" href="#carouselExampleCaptions" role="button" data-slide="prev">
+          <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+          <span class="sr-only">Previous</span>
+        </a>
+        <a class="carousel-control-next" href="#carouselExampleCaptions" role="button" data-slide="next">
+          <span class="carousel-control-next-icon" aria-hidden="true"></span>
+          <span class="sr-only">Next</span>
+        </a>
+      </div>    
+
+   </div>
+   <!-- container 3  --> 
+   <div class="container-fluid container3">
+       <div class="row">
+            <div class="col-sm-4 homewidget container3-div">
+                <!--<span><i class="glyphicon glyphicon-glass"></i></i>  </span> Not working...-->
+                <img src="./img/PngItem_2634452.png" alt="white circle">
+                <h3>Web design</h3>
+                <p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum vehicula erat non congue cursus. Aenean auctor nulla quis augue dictum, sed sagittis odio varius</p>
+            </div>
+            <div class="col-sm-4 homewidget container3-div">
+                <!--<span><i class="glyphicon glyphicon-star"></i></span>Not working...-->
+                <img src="./img/PngItem_2634452.png" alt="white circle">
+                <h3>Web development</h3>
+                <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nspendisse at erat luctus.</p>
+            </div>
+            <div class="col-sm-4 homewidget container3-div">
+                <!--<span><i class="glyphicon glyphicon-heart"></i></span>Not working...-->
+                <img src="./img/PngItem_2634452.png" alt="white circle">
+                <h3>User interface</h3>
+                <p>Cras aliquet venenatis porttitor. Donec rutrum sapien et sapien dignissim, eu dapibus mi molestie. Suspendisse dictum convallis quam et sodales. Praesent non enim et magna congue gravida.</p>
+            </div>
+        </div>
+   </div>
+   <!-- container 4  --> 
```

Comment:
Both latest articles and latest projects are more centered in the original.
Could a same class be applied to these portfolio sections to apply a margin?